src/video/wiiu/SDL_wiiuvideo.c

```diff
@@ -101,6 +101,9 @@ static int WIIU_ForegroundAcquired(_THIS)
 	GX2Invalidate(GX2_INVALIDATE_MODE_CPU, videodata->drcScanBuffer, videodata->drcScanBufferSize);
 	GX2SetDRCBuffer(videodata->drcScanBuffer, videodata->drcScanBufferSize, videodata->drcRenderMode, GX2_SURFACE_FORMAT_UNORM_R8_G8_B8_A8, GX2_BUFFERING_MODE_DOUBLE);
 
+	SDL_SendAppEvent(SDL_APP_WILLENTERFOREGROUND);
```

Comment:
If we end up deciding on only sending the enter background events if SDL handles ProcUI, it probably makes sense to put these in a `handleProcUI` check.